### DISCUSSION TITLE
Add `--no-error-on-unmatched-pattern` option to CLI

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [rome/tools#4580] [Suggestions]
Running [lintstaged](https://github.com/okonet/lint-staged) as commit hook with the following config fails with ___No files were processed in the specified paths___ when no files supported by the rome formatter has been committed.

```json
{
	"*.{js,cjs,mjs,ts,json}": ["rome format --write", "rome check"],
}
```

Rome should provide an option to ignore this error. 
For instance: 

- [Prettier](https://prettier.io/docs/en/cli.html#--no-error-on-unmatched-pattern) and [eslint](https://eslint.org/docs/latest/use/command-line-interface#--no-error-on-unmatched-pattern)  have `--no-error-on-unmatched-pattern` option
- [stylelint](https://eslint.org/docs/latest/use/command-line-interface#--no-error-on-unmatched-pattern) has `--allow-empty-input, --aei` option

**Comment by ematipico** [comment]
Issue: https://github.com/rome/tools/issues/4621
